pom.xml

```diff
@@ -280,9 +281,10 @@
             <module>plugins/jdg64</module>
             <module>plugins/jdg65</module>
             <module>plugins/jdg66</module>
+            <module>plugins/jdg70</module>
          </modules>
          <properties>
-            <plugins.jdg>jdg60,jdg61,jdg62,jdg63,jdg64,jdg65,jdg66</plugins.jdg>
+            <plugins.jdg>jdg60,jdg61,jdg62,jdg63,jdg64,jdg65,jdg66,jdg70</plugins.jdg>
```

Comment:
OK, good point. I am also getting this exception trying to run a benchmark with the plugin in embedded mode:

```
15:48:29,274 ERROR [org.radargun.stages.lifecycle.ServiceStartStage] (sc-main) Issues while instantiating/starting cache wrapper
java.lang.RuntimeException: org.infinispan.manager.EmbeddedCacheManagerStartupException: org.infinispan.commons.CacheException: Unable to invoke method public void org.infinispan.remoting.transport.jgroups.JGroupsTransport.start() on object of type HookedJGroupsTransport
	at org.radargun.service.InfinispanLifecycle.start(InfinispanLifecycle.java:61)
	at org.radargun.service.InfinispanKillableLifecycle.start(InfinispanKillableLifecycle.java:51)
	at org.radargun.stages.lifecycle.LifecycleHelper.start(LifecycleHelper.java:59)
	at org.radargun.stages.lifecycle.ServiceStartStage.executeOnSlave(ServiceStartStage.java:86)
	at org.radargun.SlaveBase.scenarioLoop(SlaveBase.java:87)
	at org.radargun.SlaveBase$ScenarioRunner.run(SlaveBase.java:151)
Caused by: org.infinispan.manager.EmbeddedCacheManagerStartupException: org.infinispan.commons.CacheException: Unable to invoke method public void org.infinispan.remoting.transport.jgroups.JGroupsTransport.start() on object of type HookedJGroupsTransport
	at org.infinispan.factories.GlobalComponentRegistry.start(GlobalComponentRegistry.java:243)
	at org.infinispan.manager.DefaultCacheManager.start(DefaultCacheManager.java:663)
	at org.radargun.service.InfinispanEmbeddedService.startCaches(InfinispanEmbeddedService.java:114)
	at org.radargun.service.Infinispan51EmbeddedService.startCaches(Infinispan51EmbeddedService.java:100)
	at org.radargun.service.InfinispanLifecycle.start(InfinispanLifecycle.java:45)
	... 5 more
Caused by: org.infinispan.commons.CacheException: Unable to invoke method public void org.infinispan.remoting.transport.jgroups.JGroupsTransport.start() on object of type HookedJGroupsTransport
	at org.infinispan.commons.util.ReflectionUtil.invokeAccessibly(ReflectionUtil.java:172)
	at org.infinispan.factories.AbstractComponentRegistry$PrioritizedMethod.invoke(AbstractComponentRegistry.java:864)
	at org.infinispan.factories.AbstractComponentRegistry.invokeStartMethods(AbstractComponentRegistry.java:633)
	at org.infinispan.factories.AbstractComponentRegistry.internalStart(AbstractComponentRegistry.java:622)
	at org.infinispan.factories.AbstractComponentRegistry.start(AbstractComponentRegistry.java:547)
	at org.infinispan.factories.GlobalComponentRegistry.start(GlobalComponentRegistry.java:231)
	... 9 more
Caused by: java.lang.NoClassDefFoundError: org/apache/logging/log4j/core/Appender
	at java.lang.ClassLoader.defineClass1(Native Method)
	at java.lang.ClassLoader.defineClass(ClassLoader.java:763)
	at java.security.SecureClassLoader.defineClass(SecureClassLoader.java:142)
	at java.net.URLClassLoader.defineClass(URLClassLoader.java:467)
	at java.net.URLClassLoader.access$100(URLClassLoader.java:73)
	at java.net.URLClassLoader$1.run(URLClassLoader.java:368)
	at java.net.URLClassLoader$1.run(URLClassLoader.java:362)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.net.URLClassLoader.findClass(URLClassLoader.java:361)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:331)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:411)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	at org.apache.logging.log4j.core.config.plugins.util.PluginRegistry.decodeCacheFiles(PluginRegistry.java:181)
	at org.apache.logging.log4j.core.config.plugins.util.PluginRegistry.loadFromMainClassLoader(PluginRegistry.java:119)
	at org.apache.logging.log4j.core.config.plugins.util.PluginManager.collectPlugins(PluginManager.java:132)
	at org.apache.logging.log4j.core.pattern.PatternParser.<init>(PatternParser.java:131)
	at org.apache.logging.log4j.core.pattern.PatternParser.<init>(PatternParser.java:112)
	at org.apache.logging.log4j.core.layout.PatternLayout.createPatternParser(PatternLayout.java:220)
	at org.apache.logging.log4j.core.layout.PatternLayout.<init>(PatternLayout.java:138)
	at org.apache.logging.log4j.core.layout.PatternLayout.<init>(PatternLayout.java:57)
	at org.apache.logging.log4j.core.layout.PatternLayout$Builder.build(PatternLayout.java:446)
	at org.apache.logging.log4j.core.config.AbstractConfiguration.setToDefault(AbstractConfiguration.java:518)
	at org.apache.logging.log4j.core.config.DefaultConfiguration.<init>(DefaultConfiguration.java:49)
	at org.apache.logging.log4j.core.LoggerContext.<init>(LoggerContext.java:75)
	at org.apache.logging.log4j.core.selector.ClassLoaderContextSelector.createContext(ClassLoaderContextSelector.java:171)
	at org.apache.logging.log4j.core.selector.ClassLoaderContextSelector.locateContext(ClassLoaderContextSelector.java:145)
	at org.apache.logging.log4j.core.selector.ClassLoaderContextSelector.getContext(ClassLoaderContextSelector.java:70)
	at org.apache.logging.log4j.core.selector.ClassLoaderContextSelector.getContext(ClassLoaderContextSelector.java:57)
	at org.apache.logging.log4j.core.impl.Log4jContextFactory.getContext(Log4jContextFactory.java:140)
	at org.apache.logging.log4j.core.impl.Log4jContextFactory.getContext(Log4jContextFactory.java:41)
	at org.apache.logging.log4j.LogManager.getContext(LogManager.java:182)
	at org.apache.logging.log4j.LogManager.getLogger(LogManager.java:471)
	at org.apache.logging.log4j.LogManager.getFormatterLogger(LogManager.java:356)
	at org.jgroups.logging.Log4J2LogImpl.<init>(Log4J2LogImpl.java:26)
	at org.jgroups.logging.LogFactory.getLog(LogFactory.java:77)
	at org.jgroups.conf.XmlConfigurator.<clinit>(XmlConfigurator.java:35)
	at org.jgroups.conf.ConfiguratorFactory.getStackConfigurator(ConfiguratorFactory.java:62)
	at org.jgroups.JChannel.<init>(JChannel.java:129)
	at org.infinispan.remoting.transport.jgroups.JGroupsTransport.buildChannel(JGroupsTransport.java:415)
	at org.infinispan.remoting.transport.jgroups.JGroupsTransport.initChannel(JGroupsTransport.java:316)
	at org.infinispan.remoting.transport.jgroups.JGroupsTransport.initChannelAndRPCDispatcher(JGroupsTransport.java:362)
	at org.infinispan.remoting.transport.jgroups.JGroupsTransport.start(JGroupsTransport.java:190)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.infinispan.commons.util.ReflectionUtil.invokeAccessibly(ReflectionUtil.java:168)
	... 14 more
Caused by: java.lang.ClassNotFoundException: org.apache.logging.log4j.core.Appender
	at java.net.URLClassLoader.findClass(URLClassLoader.java:381)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:331)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	... 62 more
```